vllm/inputs/registry.py

```diff
@@ -106,7 +107,7 @@ def _default_dummy_data_factory(
         # Avoid circular import
         from vllm.sequence import SequenceData
 
-        dummy_seq_data = SequenceData([0] * seq_len)
+        dummy_seq_data = SequenceData(array("I", [0] * seq_len))
```

Comment:
This is to reduce the msg size ("I" is half size of "l" which was the default)

Reply:
nit: The following code is faster
```suggestion
        dummy_seq_data = SequenceData(array("I", [0]) * seq_len)
```

Reply:
interesting. fixed

Reply:
fixed everything to the same way